src/main.ts

```diff
@@ -250,7 +250,6 @@ export class Purchases {
     }
     this.eventsTracker = new EventsTracker({
       apiKey: this._API_KEY,
-      httpConfig: httpConfig,
```

Comment:
I think it makes sense... Not sure if we would want to be able to inject headers in the future... but for now, this looks good!

Reply:
That is actually one of the questions I had for you. Do we want a configurable override for this? I could add two new fields to the existing `HttpConfig` such as `analyticsProxyUrl`, and `analyticsAdditionalHeaders`.